.dumi/components/SemanticPreview.tsx

```diff
@@ -134,11 +134,29 @@ const SemanticPreview: React.FC<SemanticPreviewProps> = (props) => {
     };
   }, [hoverSemantic]);
 
+  const hoveredSemanticClassNames = React.useMemo(() => {
+    if (!hoverSemantic) {
+      return semanticClassNames;
+    }
+
+    const clone = { ...semanticClassNames };
+    clone[hoverSemantic] = classnames(clone[hoverSemantic], getMarkClassName('active'));
+
+    return clone;
+  }, [semanticClassNames, hoverSemantic]);
+
   // ======================== Render ========================
+  const cloneNode = React.cloneElement(children, {
+    classNames: hoveredSemanticClassNames,
+  });
+
   return (
     <div className={classnames(styles.container)} ref={containerRef}>
       <Row style={{ minHeight: height }}>
-        <Col span={16} className={classnames(styles.colWrap)}>
+        <Col
+          span={16}
+          className={classnames(styles.colWrap, padding === false && styles.colWrapPaddingLess)}
```

Comment:
咋感觉不如传入一个 paddingLess 为true的好看一点。
```suggestion
          className={classnames(styles.colWrap, paddingLess && styles.colWrapPaddingLess)}
```